oshdb/src/test/java/org/heigit/ohsome/oshdb/grid/GridOSHNodesTest.java

```diff
@@ -18,17 +19,15 @@ public void testToString() throws IOException {
     for (int i = 0; i < 3; i++) {
       List<OSMNode> versions = new ArrayList<>();
       versions.add(new OSMNode(123L + 10 * i, 1, 123001L + 10 * i, 0L, 123, new int[] {},
-          86809727L - 1000000 * i, 494094984L - 1000000 * i));
+          86809727 - 1000000 * i, 494094984 - 1000000 * i));
       versions.add(new OSMNode(123L + 10 * i, 2, 123002L + 10 * i, 0L, 123, new int[] {},
-          86809727L - 1000000 * i, 494094984L - 1000000 * i));
+          86809727 - 1000000 * i, 494094984 - 1000000 * i));
       hosmNodes.add(OSHNodeImpl.build(versions));
     }
 
     GridOSHNodes instance = GridOSHNodes.rebase(2, 2, 100, 100000L, 86000000, 490000000, hosmNodes);
-    String expResult =
-        "Grid-Cell of OSHNodes ID:2 Level:2 BBox:(-90.000000,0.000000),(-0.000000,90.000000)";
-    String result = instance.toString();
-    assertEquals(expResult, result);
+    var entities = instance.getEntities();
+    assertEquals(hosmNodes.size(), Iterables.size(entities));
```

Comment:
maybe rename this test? because it (`testToString`) doesn't actually test the toString method anymore, does it?

Reply:
good point!

Reply:
this is not yet fixed, right?

Reply:
hmm, strange, because I thought I change the name of the test in this commit 4c19dbf7f1d2d69afb5402c8e7457bfb17e48623

Reply:
sorry, perhaps I had an old version in my browser cache or something. :shrug: